pmm/pmm.go

```diff
@@ -552,6 +553,55 @@ func (a *Admin) checkSSLCertificate() error {
 	return generateSSLCertificate(a.Config.ClientAddress, SSLCertFile, SSLKeyFile)
 }
 
+// ServerPing is a class for data from SERVER_URL/ping.
+type ServerPing struct {
+	Version string
+}
+
+// CheckVersion check server and client versions.
+func (a *Admin) CheckVersion() (bool, error) {
+	scheme := "http"
+	insecureTransport := &http.Transport{}
+	if a.Config.ServerInsecureSSL {
+		scheme = "https"
+		insecureTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	}
+	if a.Config.ServerSSL {
+		scheme = "https"
+	}
+
+	serverURL := fmt.Sprintf("%s://%s", scheme, a.Config.ServerAddress)
+	pingURL := a.qanAPI.URL(serverURL, "ping")
+	if resp, body, err := a.qanAPI.Get(pingURL); err == nil && resp.StatusCode == http.StatusOK {
+		var ping ServerPing
+		err := json.Unmarshal(body, &ping)
+		if err != nil {
+			cleanedErr := strings.Replace(err.Error(), a.serverURL, serverURL, -1)
+			return true, fmt.Errorf(`Unable to connect to PMM server by address: %s
+%s
+
+* Check if the configured address is correct.
+* If server is running on non-default port, ensure it was specified along with the address.
+* If server is enabled for SSL or self-signed SSL, enable the corresponding option.
+* You may also check the firewall settings.`, a.Config.ServerAddress, cleanedErr)
+		}
+		if Version > ping.Version {
```

Comment:
Please use https://github.com/percona/pmm/blob/master/version/version_info.go. The same code will be used in 2.0.